From b0fca81f9168e2331bc79f7127b80b711ae6c076 Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Thu, 23 Feb 2023 10:26:39 +0100
Subject: [PATCH] rework: Integrate machine independent resin environment
 configuration

This patch integrates resin default environment configuration in an u-boot which
has a version with Kconfig support.

Signed-off-by: Andrei Gherzan <andrei@resin.io>

Upstream-Status: Inappropriate [configuration]
---
 include/env_default.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/env_default.h b/include/env_default.h
index ea31a8eddf..660a86e911 100644
--- a/include/env_default.h
+++ b/include/env_default.h
@@ -9,6 +9,7 @@
 
 #include <env_callback.h>
 #include <linux/stringify.h>
+#include <env_resin.h>
 
 #ifdef DEFAULT_ENV_INSTANCE_EMBEDDED
 env_t embedded_environment __UBOOT_ENV_SECTION__(environment) = {
@@ -28,6 +29,7 @@ const uchar default_environment[] = {
 #ifdef	CONFIG_ENV_CALLBACK_LIST_DEFAULT
 	ENV_CALLBACK_VAR "=" CONFIG_ENV_CALLBACK_LIST_DEFAULT "\0"
 #endif
+    BALENA_ENV
 #ifdef	CONFIG_ENV_FLAGS_LIST_DEFAULT
 	ENV_FLAGS_VAR "=" CONFIG_ENV_FLAGS_LIST_DEFAULT "\0"
 #endif
-- 
2.37.2

